docs/user/debugging.md

```diff
@@ -4,154 +4,243 @@
+* Successful tasks: 16
+* Task Count: 16
+* Queued tasks: 0
+* Evidence Name: GoogleCloudDisk:processed-disk-name1
+* Evidence ID: None
+
+## Request ID: 63de1268000f97d85198949e3fdb8dk5
+* Last Update: 2024-04-13T01:45:33.141508Z
+* Requester: turbiniauser
+* Reason:
+* Status: completed_with_errors
+* Failed tasks: 2
+* Running tasks: 0
+* Successful tasks: 14
+* Task Count: 16
+* Queued tasks: 0
+* Evidence Name: GoogleCloudDisk:processed-disk-name2
+* Evidence ID: 79fa702fe21a476f8a560dab4fcdae23
+```
 
+The above output shows that there were two requests recently executed, but
+doesn't show anything about the Tasks associated with those requests and now
+that we have the request IDs we can use that to find more info.
+`turbinia-client status request <req id>` will get the details for the given
+request ID and the associated Tasks that were generated to process the input
+Evidence.
 ```
-$ ./turbiniactl -a status -d1
-
-Retrieved 2 Task results:
-2017-12-06T15:09:15.013Z request: None task: 1f8c4b321f444614bde296b0a00bb91f PsortTask Failed: Execution failed with status 1
-        gs://my-turbinia-bucket/output/1512601742-1f8c4b321f444614bde296b0a00bb91f-PsortTask/worker-log.txt
-2017-12-06T15:08:49.616Z request: b94ad420cf5a483fb3fc4409d5fc6dcd task: 020fe728a2b64dc7ba1da2656e1cc454 PlasoTask Successful: Completed successfully in 0:00:14.178579 on aaronp.dev
-        gs://my-turbinia-bucket/output/1512601704-020fe728a2b64dc7ba1da2656e1cc454-PlasoTask/020fe728a2b64dc7ba1da2656e1cc454.log
-        gs://my-turbinia-bucket/output/1512601704-020fe728a2b64dc7ba1da2656e1cc454-PlasoTask/worker-log.txt
+$ turbinia-client status request 63de1268000f97d85198949e3fdb8dk5
+
+## Request ID: 63de1268000f97d85198949e3fdb8dk5
+* Last Update: 2024-04-13T01:45:33.141508Z
+* Requester: turbiniauser
+* Reason:
+* Status: completed_with_errors
+* Failed tasks: 2
+* Running tasks: 0
+* Successful tasks: 14
+* Task Count: 16
+* Queued tasks: 0
+* Evidence Name: GoogleCloudDisk:processed-disk-name2
+* Evidence ID: 79fa702fe21a476f8a560dab4fcdae23
+
+[...snip other Task details...]
+
+### PsortTask (MEDIUM PRIORITY): Execution failed with status 1
+### PlasoParserTask (MEDIUM PRIORITY): Completed successfully in 0:00:14.178579 on turbinia-worker-1
 ```
 
-Note that there may be local paths that show up in the
-`turbiniactl status` output, and these are local to the Worker that
-executed that Task.
+In the example output above we can see that there was a failure of the
+`PsortTask`, but we don't have any other further details because by default it
+will only show those details for Tasks with high priority findings in order to
+keep the output manageable.
 
-Using the saved paths above we can display the output with `gsutil`.
+To show more details for Tasks with lower priorities we can set the priority
+filter with `--priority_filter` or `-p` and to see all tasks we can set `-p 100`
+(priorities range from `0` to `100` where priority `0` is the highest and `100`
```

Comment:
Maybe the parentheses just need to move up to my other comment about when you initially mention priorities

Reply:
Done